app/portainer/components/datatables/components/ExpandingCell.tsx

```diff
@@ -0,0 +1,26 @@
+import { PropsWithChildren } from 'react';
+import { Row, TableInstance } from 'react-table';
+
+interface Props {
+  row: Row<TableInstance>;
+}
+
+export function ExpandingCell({ children, row }: PropsWithChildren<Props>) {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I would prefer if you would state them explicitly like you do in the datatable

Reply:
I tried to as well, but get typescript errors when trying to use `onClick`, something very weird, because I can event see it's value if I console.log it